aiida_vasp/backendtests/basic.py

```diff
@@ -105,19 +105,24 @@ def test_write_kpoints_mesh(self):
         calc = self._get_calc('c', 'm')
         inp = calc.get_inputs_dict()
         calc.write_kpoints(inp, self.tmpf)
-        kpp = KpParser(self.tmpf)
-        self.assertTrue((kpp.kpoints == Common.kpoints_mesh_res()).all())
+        kpp = KpParser(self.tmpf, 'KPOINTS')
```

Comment:
The backendtests/basic.py are old and will not be run on CI!

Please migrate this test to aiida_vasp/io/tests/test_kpoints_io.py (This will also increase coverage)